python/openai/README.md

```diff
@@ -34,6 +34,13 @@
 
 ## Pre-requisites
 
+> [!WARNING]
+> **CuPy CUDA 13 Compatibility Issue**: The Triton Inference Server Image has been upgraded to CUDA 13. The latest CuPy wheel released at the time of writing is not fully compatible with CUDA 13. If you encounter issues with CuPy, they may be fixed by running the following commands before starting the Frontend:
+> ```bash
+> ln -sf /usr/local/cuda/targets/x86_64-linux/lib/libnvrtc.so.13.0.48 /usr/local/cuda/targets/x86_64-linux/lib/libnvrtc.so.12
```

Comment:
This probably won't work if job using compiler operations. 
Compiler will look for `libnvrtc.so.12` SONAME in order to resolve dependency but library `libnvrtc.so.13.0.48` will have name `libnvrtc.so.13`: 
```
root@ad4f880c5b30:/opt/tritonserver/qa/L0_batcher_cudashm# readelf -a /usr/local/cuda-13.0/targets/x86_64-linux/lib/libnvrtc.so.13.0.48 | grep SON
 0x000000000000000e (SONAME)             Library soname: [libnvrtc.so.13]
```

Reply:
We don't compile CuPy, it comes from a pip wheel installed only when we are installing tritonfrontend wheel. We just need to make sure it links during runtime and that is sufficient here.

Reply:
Can you apply the fix?